service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java

```diff
@@ -173,9 +173,9 @@ public ResponseEntity<?> generate(final String msgProtocol, final String msgType
                 int failedCount = 0;
                 for (JsonElement element : inputEventJsonArray) {
                     try {
-                        JsonObject generatedEvent = (processEvent(msgProtocol, msgType,
+                        JsonObject generatedEvent = processEvent(msgProtocol, msgType,
```

Comment:
processEvent should be renamed to generateEvent to be more descriptive.

Reply:
Done